src/docs/style/logo.mdx

```diff
@@ -0,0 +1,267 @@
+        `3: Favicon for web apps, previewed at a high resolution.`,
+        `4: Amazon Alexa Skills' rendering.`,
+        `5 – 8: Android's adaptive icon allows phone manufacturers to render app logos in different shapes.`,
+    ]}
+    regularWidth
+/>
+
+Different platforms have different product logo requirements — they often have different shapes, resolutions, safe areas, and parallax layers. Click on the links below to learn more about logo guidelines on each platform.
+
+-   [Apple (iOS, iPadOS, macOS, tvOS, watchOS)](https://developer.apple.com/design/human-interface-guidelines/foundations/app-icons/)
+-   Android
+    -   [Adaptive Icons](https://developer.android.com/develop/ui/views/launch/icon_design_adaptive)
+        -   [Image Asset Studio](https://developer.android.com/studio/write/image-asset-studio.html) in Android Studio
+        -   Adaptive Icon [Design Guide](https://medium.com/google-design/designing-adaptive-icons-515af294c783)
+    -   [Google Play](https://developer.android.com/distribute/google-play/resources/icon-design-specifications)
+-   [Windows](https://learn.microsoft.com/en-us/windows/apps/design/style/iconography/app-icon-design)
+
+## Favicons
+
+Favicons are product logos for web apps. They can appear in browser tabs, task bars, bookmarks, and phone screens just like native mobile apps, under various image sizes. We highly recommend redesigning the app logo specifically for the favicon to ensure that it shows well at all resolutions.
+
+According to [Mailchimp](https://mailchimp.com/resources/favicon-size/), the optimal favicon dimensions are as follows, in pixels:
+
+-   Browser tab favicons - 16×16
+-   Taskbar shortcut icons - 32×32
+-   Desktop shortcut icons - 96×96
+-   Apple touch icons - 180×180
+
+Developers should [include multiple favicons](https://blog.hubspot.com/website/what-is-a-favicon) for these different resolutions in the HTML head to accommodate for different favicon usage.
+
+Favicons used in browser tabs and taskbar shortcuts resemble the original product logo design, but with the badge removed. It always has the background color to ensure that the glyph can be seen clearly.
+
+<ImageGrid
+    images={[Favicon]}
+    caption={[
+        `Favicon needs to be recognizable at a size as small as 16×16 pixels.`,
+        `1: The original product logo, with the bottom badge removed to allow more room for the glyph.`,
+        `2: The same logo, rendered at 16×16, a favicon size used in browser tabs.`,
+        `3: Optionally, you can consider doing "pixel-hinting" — manually editing the favicon to ensure that it renders well at low resolution.`,
+        `4: The favicon in a browser tab.`,
+    ]}
+    regularWidth
+/>
+
+Desktop shortcuts and Apple touch icons should resemble the design of standard app logos.
+
+# Design Specifications
+
+The templates should already have everything you need. However, if you want to do something fancy, here's the design spec for a default logo:
```

Comment:
```suggestion
The provided templates have implemented these specifications for you already, but they are presented here for your reference if you need to make any customizations:
```

Reply:
thanks! I definitely needed this one... I really don't know how to express it politely, but you nailed it 😂